tiptapy/__init__.py

```diff
@@ -89,7 +89,10 @@ def inner_render(self, node) -> str:
             caption = attrs.get('caption', '').strip()
             if caption:
                 html += f"<figcaption>{caption}</figcaption>"
-        return f'<div class="video-wrapper"><figure>{html}</figure></div>'
+        provider_name = attrs.get('provider', 'link')
```

Comment:
@Pradhvan please add test case for unknown provider.

Reply:
Well spotted @gauravr